qcore/validate_vm.py

```diff
@@ -103,17 +106,42 @@ def vmfile(filename):
 #        # not as important, can be re-created based on params_vel.py
 #        pass
 
+    # 8: Check VM within bounds - only if DEM file is provided. If DEM file is not present, does not return false
+    if dem_path is None:
+        dem_path = DEM_PATH
+    if os.path.exists(dem_path):
```

Comment:
Comment above says that it only checks if there is a DEM file provided. I guess I can make it fail otherwise. But didn't want to break existing functionality with a new requriment.

Reply:
I didn't read the comment above..., I just saw that it would still return True and a happy message if the file doesn't exist, so that didn't seem right to me. But given the comment, there is probably (hopefully) a reason to return True if the file doesn't exist. So maybe just leave it as you have it?